src/EditorFeatures/Core.Wpf/InlineDiagnostics/AbstractDiagnosticsTaggerProvider.SingleDiagnosticKindPullTaggerProvider.cs

```diff
@@ -43,7 +43,13 @@ private sealed class SingleDiagnosticKindPullTaggerProvider(
         IDiagnosticAnalyzerService analyzerService,
         IGlobalOptionService globalOptions,
         ITextBufferVisibilityTracker? visibilityTracker,
-        IAsynchronousOperationListener listener) : AsynchronousTaggerProvider<TTag>(threadingContext, globalOptions, visibilityTracker, listener)
+        IAsynchronousOperationListenerProvider listenerProvider,
+        string featureName) : AsynchronousTaggerProvider<TTag>(
+            threadingContext,
+            globalOptions,
+            visibilityTracker,
+            listenerProvider,
+            featureName)
```

Comment:
all the taggers now pass the listenerProvider and feature name down.  thsi is because the base type actually wants access to the listenerProvider itself.